lib/Completion/Bridge/TolerantParser/WorseReflection/WorseClassMemberCompletor.php

```diff
@@ -81,7 +85,7 @@ public function complete(Node $node, TextDocument $source, ByteOffset $offset):
 
         foreach ($type->expandTypes()->classLike() as $type) {
             foreach ($this->populateSuggestions($nodeContext, $type, $static, $shouldCompleteOnlyName, $isInstance) as $suggestion) {
-                if ($partialMatch && 0 !== mb_strpos($suggestion->name(), $partialMatch)) {
+                if ($partialMatch && !str_starts_with($suggestion->name(), $partialMatch)) {
```

Comment:
Generally, don't need the `mb_` version of the function for this purpose. Look at <https://wiki.php.net/rfc/str_contains#case-insensitivity_and_multibyte_strings>.